xray/commands/audit/commonutils.go

```diff
@@ -69,27 +70,35 @@ func (auditCmd *AuditCommand) runScanGraph(modulesDependencyTrees []*services.Gr
 	}
 	var results []services.ScanResponse
 	for _, moduleDependencyTree := range modulesDependencyTrees {
-		params := &services.XrayGraphScanParams{
+		params := services.XrayGraphScanParams{
 			Graph:      moduleDependencyTree,
 			RepoPath:   auditCmd.targetRepoPath,
 			Watches:    auditCmd.watches,
 			ProjectKey: auditCmd.projectKey,
+			ScanType:   services.Dependency,
 		}
 
 		// Log the scanned module ID
-		log.Info("Scanning module " + moduleDependencyTree.Id[strings.Index(moduleDependencyTree.Id, "//")+2:] + "...")
+		moduleName := moduleDependencyTree.Id[strings.Index(moduleDependencyTree.Id, "//")+2:]
+		log.Info("Scanning module " + moduleName + "...")
 
 		// Scan and wait for results
-		scanId, err := xrayManager.ScanGraph(*params)
+		scanId, err := xrayManager.ScanGraph(params)
 		if err != nil {
 			return err
 		}
 		scanResults, err := xrayManager.GetScanGraphResults(scanId, auditCmd.includeVulnerabilities, auditCmd.includeLicenses)
 		if err != nil {
 			return err
 		}
-		results = append(results, *scanResults)
+		scanErr := commands.CheckScanResultsStatus(scanResults.ScannedStatus , moduleName)
+		if scanErr == nil {
+			results = append(results, *scanResults)
+		}
```

Comment:
I this that if scanErr is not nil, we should add an error log message with the error string. Otherwise, no one will ever know that scanning this component failed.